commands/remove.go

```diff
@@ -14,7 +14,6 @@ import (
 
 func init() {
 	// Setup flags that are used by multiple commands (variables defined in faas.go)
-	removeCmd.Flags().StringVar(&functionName, "name", "", "Name of the deployed function")
 	removeCmd.Flags().StringVar(&gateway, "gateway", defaultGateway, "Gateway URI")
```

Comment:
`--name` no longer used? Why the removal?

Reply:
--name is never used

This command uses positional argument

```
 	Use: `remove FUNCTION_NAME [--gateway GATEWAY_URL]
---
 		if len(args) < 1 {
 			fmt.Println("Please provide the name of a function to delete")
 			return
 		}
```

Reply:
I'm not sure about this.. it looks unintentional @johnmccabe .. we have flag based elsewhere.. shouldn't we change this to use --name too?

Reply:
Positional argument, in my opinion, is more suitable for this type of command (invoke, remove). Less characters to type and in case we implement multiple names, the command is more easily to use

`faas-cli remove FUNC1 FUNC2 FUNC3`
vs
`faas-cli remove --name FUNC1 --name FUNC2 --name FUNC3`

In the case we use -f, --name can be replaced with --filter easily.